solutions/ha-nva/layer-7-ingress/README.md

```diff
@@ -0,0 +1,119 @@
+# Highly Available Ingress with layer 7 NVAs
+
+This sample template will deploy an Azure Application Gateway in front of two simulated network virtual appliance (NVA) VMs.  The NVA VMs will forward requests on port 80 to a Web-tier Azure Load Balancer, which will distribute traffic across two back-end VMs running a simple web server.
+
+For more information on this deployment and other highly available NVA options, see [Deploy highly available NVAs](https://docs.microsoft.com/azure/architecture/reference-architectures/dmz/nva-ha).
+
+![Highly Available Ingress with layer 7 NVAs architectural diagram.](l7-ingress.png)
+
+This sample architecture leverages concepts from the [Example Parameterized Deployment With Linked Templates](https://azure.microsoft.com/resources/templates/301-parameterized-linked-templates/) and [Create ssh-keys and store in Key Vault](https://azure.microsoft.com/resources/templates/201-deployment-script-ssh-key-gen/) samples.
+
+## Deploy this template to Azure
```

Comment:
Couple thoughts on this section

- I have been using 'reference deployment' as the title for these sections. Should we standardize? 
- I'm wondering if we can remove the copy around forking and instead including that in the code readme?

Check out hub and spoke for examples. If you think there is a better way, let me know, and we can retro-fill this back to the completed ones.